fvm/blueprints/epochs.go

```diff
@@ -1,13 +1,275 @@
 package blueprints
 
 import (
+	"encoding/hex"
+	"fmt"
+
 	"github.com/onflow/cadence"
 	jsoncdc "github.com/onflow/cadence/encoding/json"
 	"github.com/onflow/flow-core-contracts/lib/go/templates"
 
+	"github.com/onflow/flow-go/model/bootstrap"
+
+	"github.com/onflow/flow-go/module/epochs"
+
 	"github.com/onflow/flow-go/model/flow"
 )
 
```

Comment:
Is is possible to use templates from the core contracts repo and SDK here instead of hard-coding the transaction text?

Reply:
It looks like most of the hardcoded transactions either don't have a corresponding tx in **_templates_** or they are different slightly.

Reply:
that could mean it might be worth adding some of them, but I guess that make sense